File: 0125-valid-palindrome/0125-valid-palindrome.py
class Solution(object):
    def isPalindrome(self, s):
        """
        :type s: str
        :rtype: bool
        """
        s = s.strip().lower()
        string = ""
        for i in range(len(s)):
            code = ord(s[i])
            if(code >= 97 and code <= 122) or (code >= 48 and code <= 57):
                string = string + s[i]

        reverse = ""
        for i in range(len(string)-1,-1,-1):
            reverse = reverse + string[i]
        return string == reverse
        

File: 0125-valid-palindrome/0125-valid-palindrome.js
/**
 * @param {string} s
 * @return {boolean}
 */
var isPalindrome = function (s) {
    s = s.trim().toLowerCase();
    let str = ""
    for (let i = 0; i < s.length; i++) {
        let code = s.charCodeAt(i);
        if ((code >= 97 && code <= 122) || (code >= 48 && code <= 57)) {
            str = str + s[i]
        }
    }
    let reverse = "";
    for (let i = str.length - 1; i >= 0; i--) {
        reverse = reverse + str[i]
    }
    return str === reverse;
};